test/unit/com/unifina/service/UserServiceSpec.groovy

```diff
@@ -68,7 +68,7 @@ class UserServiceSpec extends Specification {
 	def "the user is created when called, with default roles if none supplied"() {
 		when:
 		createData()
-		SecUser user = service.createUser([username: "test@test.com", name:"test", password: "test", enabled:true, accountLocked:false, passwordExpired:false])
+		SecUser user = service.createUser([username: "test@test.com", name:"test", password: "test", enabled:true, accountLocked:false, passwordExpired:false, lastLogin: new Date()])
```

Comment:
Can remove `lastLogin` if you set the default field value in `SecUser.groovy` (also elsewhere in this file)